kube_operator/api_objects.go

```diff
@@ -588,16 +597,23 @@ func (cr CustomResourceV1) Install(c KubeClient, namespace string) error {
 
 	// the cluster has to create the endpoint for the custom resource, this can take some time
 	// the cr cannot exist without the crd so we don't have to worry about it already existing
-	glog.V(3).Infof(kwlog(fmt.Sprintf("creating operator custom resource %v", cr.CustomResourceObject)))
+	timeout := cr.InstallTimeout
```

Comment:
I'm trying to get everyone to standardize how config defaults are set. Look at the agent config: https://github.com/open-horizon/anax/blob/master/config/config.go#L297 the default is set into the config object and then the default is overridden when the config file is read in IF the real config file has a value in it.

Reply:
Fixed